examples/nau7802_test/nau7802_test.ino

```diff
@@ -7,6 +7,7 @@ void setup() {
   Serial.println("NAU7802");
   if (! nau.begin()) {
     Serial.println("Failed to find NAU7802");
+    while (true) {}  // Don't proceed.
```

Comment:
Please change this to `while (1) delay(10);`  to be more in line with other examples. The `delay()` helps prevent WDT resets on some boards.

Reply:
Your wish is granted 🪄:

https://github.com/adafruit/Adafruit_NAU7802/compare/2c289a571bc2bd409b55cbbff18ad9f9f843d40c..b3203ef2f63f9bbd13d81f8dc4135cca60298158

Reply:
thanks!